styleguide/source/_patterns/02-molecules/article-stub.md

```diff
@@ -0,0 +1,51 @@
+### Description
+An Article Stub (renamed from Article Preview) molecule contains a topic label (optional), an article title, a description, and optionally either an image or a Youtube embed and metadata that includes data and read/playtime. Variants are defined based on what fields are needed in each variant, i.e. image or video. The order of elements is determined using flexbox where different from the base pattern.
+
+Variants are managed by use of pseudo-patterns.
+
+- "Article Stub Small Image" is the variant of this pattern that includes a smaller image.
+- "Article Stub Video" is the variant of this pattern that includes a video rather than an image.
+- "Article Stub Related" is the variant of this pattern that includes an image and title.
+- "Article Stub Related No Image" is the variant of this pattern that includes a linked title.
```

Comment:
Should psudeo-patterns be using the "as [variant]" syntax for naming here?

Reply:
probably but I'd rather investigate that later/next week because we don't have time to rename the rest of the things before deploy deadline today; also we need to give more thought about what everything is named in general so I'd rather do that than make a one off change

Reply:
I've added a note to discuss the issue at the next FED meet. Please add a ticket to remediate the bug later.